core/templates/dev/head/base.html

```diff
@@ -72,34 +72,37 @@
 
     <script>
       var GLOBALS = {
-        ADDITIONAL_ANGULAR_MODULES: [],
-        csrf_token: JSON.parse('{{csrf_token|js_string}}'),
-        DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
-        MINIFICATION: JSON.parse('{{MINIFICATION|js_string}}'),
-        INVALID_NAME_CHARS: JSON.parse('{{INVALID_NAME_CHARS|js_string}}'),
         ACTIVITY_STATUS_PRIVATE: JSON.parse(
           '{{ACTIVITY_STATUS_PRIVATE|js_string}}'),
         ACTIVITY_STATUS_PUBLIC: JSON.parse(
           '{{ACTIVITY_STATUS_PUBLIC|js_string}}'),
         ACTIVITY_STATUS_PUBLICIZED: JSON.parse(
           '{{ACTIVITY_STATUS_PUBLICIZED|js_string}}'),
+        ADDITIONAL_ANGULAR_MODULES: [],
         ALL_CATEGORIES: JSON.parse('{{ALL_CATEGORIES|js_string}}'),
         ALL_LANGUAGE_CODES: JSON.parse('{{ALL_LANGUAGE_CODES|js_string}}'),
+        CAN_SEND_ANALYTICS_EVENTS: JSON.parse(
+          '{{CAN_SEND_ANALYTICS_EVENTS|js_string}}'),
+        csrf_token: JSON.parse('{{csrf_token|js_string}}'),
         DEFAULT_LANGUAGE_CODE: JSON.parse(
           '{{DEFAULT_LANGUAGE_CODE|js_string}}'),
+        DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
+        INVALID_NAME_CHARS: JSON.parse('{{INVALID_NAME_CHARS|js_string}}'),
+        MINIFICATION: JSON.parse('{{MINIFICATION|js_string}}'),
         RTE_COMPONENT_SPECS: JSON.parse('{{RTE_COMPONENT_SPECS|js_string}}'),
```

Comment:
Oh, I see. I had assumed that https://github.com/oppia/oppia/pull/2275/files#diff-f66cda1f67929310d3b56713d450f96aR93 refers to all keys declared below it. Thus, didn't mix and match. But the comment is just for `POST_COMPLETION_HOOKS`, right ?